examples/storm-kafka-client-examples/pom.xml

```diff
@@ -73,12 +73,6 @@
             <version>${project.version}</version>
             <scope>${provided.scope}</scope>
         </dependency>
-        <dependency>
```

Comment:
Why was this removed?

Reply:
I think we don't need kafka server jar to run  examples. Ran the tests without this dependency. But i reverted the change, since it is not related to this PR.